VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegment.java

```diff
@@ -111,6 +111,8 @@ Properties trackVideoOrientation(String videoId, Double currentTime,
 
     Properties courseShared(String courseId, String socialNetwork);
 
+    Properties courseDetailShared(String courseId, String category, String about_url, String type);
```

Comment:
Can we tare out the old course sharing feature? It's behind a config flag (`SOCIAL_SHARING`) which is set to false in all our configs. Do we need it?